### ISSUE TITLE
Can fitparse be used for getting HRV data from a .FIT file?

### ISSUE THREAD

**Issue by pitts85** [dtcooper/python-fitparse#98]
I'm able to get the data using Golden Cheetah so I know it's in the file.  I'd really like to use this code to do it.

**Comment by felipeam86** [comment]
I was wondering the same thing this week

**Comment by tnikkel** [comment]
If you post an example fit file someone might be able to take a look.

**Comment by tsteven4** [comment]
yes.  for example https://github.com/tsteven4/fit-analysis/blob/main/src/fit_analysis/analyze_hrv.py
